uplink/builder.py

```diff
@@ -138,14 +148,24 @@ def build(self, *args, **kwargs):
             except exceptions.InvalidRequestDefinition as error:
                 # TODO: Find a Python 2.7 compatible way to reraise
                 raise exceptions.UplinkBuilderError(
-                    self._service_cls, attribute_name, error)
-            call_factory = self._make_call_factory(service, definition)
-            self._bind_to_instance(service, attribute_name, call_factory)
-        return service
+                    self._service_stub.__class__, attribute_name, error)
+            call_factory = self._make_call_factory(self._service_stub, definition)
+            self._bind_to_instance(self._service_stub, attribute_name, call_factory)
+        return self._service_stub
+
+class Consumer(object):
```

Comment:
To support the use case example outline in #3, we'll need to add this to the builder module's __all__.